### PR TITLE
Kevin pandas

### PR DESCRIPTION
**Pull request by K-Wangaroo**
Compile master dataset (the dataset is not on git but is on the VM)

### CODE REVIEW

File: Data_Collection.ipynb [hunk 1]
```diff
@@ -0,0 +1,1759 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "id": "1e46ad30-35e9-4f27-8ace-f7f0f12c21e9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "new_df = new_df.merge(genre_df, how=\"left\", on=[\"track\", \"artist\"]).merge(lyrics_df, how=\"left\", on=[\"track\", \"artist\"])"
```
Comment (stevenwinnick) [inline Data_Collection.ipynb:1305]: Why do you have to .merge twice?

Comment (stevenwinnick) [reply Data_Collection.ipynb:1305]: Oh I see, one is the genre df and one is lyrics

**Review by stevenwinnick — COMMENTED** [review]
Nice work Kevin!

File: Data_Collection.ipynb (hunk 1 above)
Comment (sc4789) [reply Data_Collection.ipynb:1305]: The compiled dataset was organized perfectly so that we can perform feature extraction and preprocess the input!

Comment (preethiprakash1) [reply Data_Collection.ipynb:1305]: This dataset was very helpful when compiling the overall model! I was able to use what you created to form the targets.

File: Data_Collection.ipynb [hunk 2]
```diff
@@ -0,0 +1,1759 @@
...
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                   track            artist genre\n",
+       "0  Jealous Kind Of Fella     Garland Green   edm\n",
+       "1          Initials B.B.  Serge Gainsbourg   pop\n",
+       "2           Melody Twist       Lord Melody   pop\n",
+       "3          Mi Bomba Sonó        Celia Cruz   pop\n",
```
Comment (tinghui2012) [inline Data_Collection.ipynb:795]: what is the dictionary mapping defined from genre to the integer values in the dataset?

Comment (K-Wangaroo) [reply Data_Collection.ipynb:795]: didn't do any mapping here, let Preethi/Sana do it in training so they could interpret it in the model!

**Pull request merged**